recipes/_bash.rb

```diff
@@ -21,6 +21,7 @@
 include_recipe 'omnibus::_common'
 
 return if windows?
+return if solaris_10?
```

Comment:
Anywhere we return on `solaris_10?` it might be nice to add a comment mentioning that Bash ships in the omnibus build-essential package.